code/modules/mob/dead/new_player/login.dm

```diff
@@ -1,4 +1,6 @@
 /mob/dead/new_player/Login()
+	if(config.use_exp_tracking)
+		client.set_exp_from_db()

```

Comment:
>That would cause the exp to be reset to the db value every time the client reconnected.

That's the point.

We have two servers, you can't assume this data hasn't changed, it should be updated on the server when it changes, and it shouldn't be cached for too long.